tests/e2e/lib/setup-env.js

```diff
@@ -72,10 +79,19 @@ export const catchBeforeAll = async ( callback, errorHandler = defaultErrorHandl
 	} );
 };
 
+let video;
+const captureVideo = process.env.CAPTURE_VIDEO === 'true';
+
 async function setupBrowser() {
-	const userAgent = await browser.userAgent();
-	await page.setUserAgent( userAgent + ' wp-e2e-tests' );
-	await setBrowserViewport( 'large' );
+	const userAgent = await page.evaluate( () => navigator.userAgent );
+	logger.info( `User agent: ${ userAgent }` );
+
+	if ( captureVideo ) {
```

Comment:
To be honest, I don't think this belongs to `setupBrowser`. Also, since `setupBrowser` is being called in `beforeAll` and `afterEach` we end up with 28 video files.

I guess it does not directly related to the goal of this PR, but I'll still share my thoughts.

It would be nice to have a video per test, and save it only if test is failing (I don't think we successful runs as much)

Reply:
I totally agree. But also, calling `setupBrowser` in an `afterEach` doesn't sound right to me either.
I guess I already mentioned, the entire setup could do some refactoring, maybe combined with a test runner migration. Currently my feeling is that everything is all over the place. Probably a custom environment would make most sense.
And true, not in the scope of this PR. But I will find a better place for video handling.

Reply:
Solved now, see https://github.com/Automattic/jetpack/pull/17814#discussion_r529659048 

By also moving the logic of setting user agent from `setupBrowser` the method remained empty, so I removed it.